src/index.ts

```diff
@@ -66,9 +66,15 @@ export interface MandeInstance {
    * @param url - relative url to send the request to
    * @param options - optional {@link Options}
    */
-  get(url: string | number, options: Options<'response'>): Promise<Response>
-  get(url: string | number, options: Options<'text'>): Promise<string>
-  get<T = unknown>(url: string | number, options?: Options): Promise<T>
+  get(
+    url: string | number | undefined,
+    options: Options<'response'>
+  ): Promise<Response>
+  get(
+    url: string | number | undefined,
+    options: Options<'text'>
+  ): Promise<string>
+  get<T = unknown>(url?: string | number, options?: Options): Promise<T>
```

Comment:
there should also be a version that allows `get(options?: Options)`. This will also require adding some types to `delete()` BTW